test/sequential/test-module-loading.js

```diff
@@ -247,12 +247,12 @@ assert.deepEqual(children, {
 assert.throws(function() {
   console.error('require non-string');
   require({ foo: 'bar' });
-}, 'path must be a string or Buffer');
+}, /path must be a string/);
```

Comment:
This is more lenient. The test will still pass if the old message is returned right?